src/NLog/Config/LoggingRuleLevelFilter.cs

```diff
@@ -62,9 +62,13 @@ public LoggingRuleLevelFilter GetSimpleFilterForUpdate()
 
         public LoggingRuleLevelFilter SetLoggingLevels(LogLevel minLevel, LogLevel maxLevel, bool enable)
         {
-            for (int i = minLevel.Ordinal; i <= Math.Min(maxLevel.Ordinal, LogLevels.Length - 1); ++i)
+            // If minLevel and maxLevel are swapped, swap again to correct it.
```

Comment:
Could this code-change be move to another pull-requests, since it does more than improving xml-comments.